tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -191,18 +194,23 @@ def test_should_call_run_command_method_when_helm_install_with_non_defaults(
                 "test-release",
                 "test-repository/streams-app",
                 "--install",
-                "--timeout=120s",
                 "--namespace",
                 "test-namespace",
                 "--values",
                 "values.yaml",
+                "--set-file",
+                "key1=example\\path1,key2=example\\path2"
+                if platform.system() == "Windows"
+                else "key1=example/path1,key2=example/path2",
```

Comment:
I think you could use something like os.sep, but I am not sure

Reply:
good idea!